src/patternfly/_chart-globals.scss

```diff
@@ -50,3 +51,179 @@ $pf-chart-color-black-200: $pf-color-black-400;
+
+// Axis Chart
+$pf-chart-axis--axis--stroke--Width: 1;
+$pf-chart-axis--axis--stroke--Color: $pf-color-black-900;
+$pf-chart-axis--axis--fill: transparent;
+$pf-chart-axis--axis-label--Padding: 100;
+$pf-chart-axis--axis-label--stroke--Color: transparent;
+$pf-chart-axis--grid--fill: transparent;
+$pf-chart-axis--grid--stroke--Color: transparent;
+$pf-chart-axis--grid--PointerEvents: "painted";
+$pf-chart-axis--tick--fill: transparent;
+$pf-chart-axis--tick--Width: 1;
+$pf-chart-axis--tick--stroke--Color: transparent;
+$pf-chart-axis--tick-label--fill: $pf-chart-color-black-500;
+
+// Bar Chart
+$pf-chart-bar--Width: 10;
+$pf-chart-bar--data--stroke: none;
+$pf-chart-bar--data--fill: $pf-color-black-900;
+$pf-chart-bar--data--Padding: 8;
+$pf-chart-bar--data-stroke--Width: 0;
+
+// Box Plot Chart
+$pf-chart-boxplot--max--Padding: 8;
+$pf-chart-boxplot--max--stroke--Color: $pf-color-black-900;
+$pf-chart-boxplot--max--stroke--Width: 1;
+$pf-chart-boxplot--median--Padding: 8;
+$pf-chart-boxplot--median--stroke--Color: $pf-color-black-900;
+$pf-chart-boxplot--median--stroke--Width: 1;
+$pf-chart-boxplot--min--Padding: 8;
+$pf-chart-boxplot--min--stroke--Width: 1;
+$pf-chart-boxplot--min--stroke--Color: $pf-color-black-900;
+$pf-chart-boxplot--lower-quartile--Padding: 8;
+$pf-chart-boxplot--lower-quartile--fill: $pf-chart-color-black-300;
+$pf-chart-boxplot--upper-quartile--Padding: 8;
+$pf-chart-boxplot--upper-quartile--fill: $pf-chart-color-black-300;
+$pf-chart-boxplot--box--Width: 20;
+
+// Bullet Chart
+$pf-chart-bullet--Height: 55;
+$pf-chart-bullet--items-per-row: 2;
+$pf-chart-bullet--legend--Height: 30;
+$pf-chart-bullet--range--Width: 40;
+$pf-chart-bullet--threshold-error--Color: $pf-color-red-100;
+$pf-chart-bullet--threshold-error--Width: 2;
+$pf-chart-bullet--value--Width: 9;
+$pf-chart-bullet--comparative-measure-line--warning--stroke--Color: $pf-color-orange-300;
+$pf-chart-bullet--comparative-measure-line--warning--BorderWidth: 1;
+$pf-chart-bullet--comparative-measure-line--error--stroke--Color: $pf-color-red-100;
+$pf-chart-bullet--comparative-measure-line--error--BorderWidth: 1;
+$pf-chart-bullet--zero-line--stroke--Color: $pf-color-black-700;
+$pf-chart-bullet--zero-line--BorderWidth: 1;
+$pf-chart-bullet--negative-primary-measure--stroke--Color: $pf-color-red-100;
+$pf-chart-bullet--negative-primary-measure--BorderWidth: 2;
+
+// Candlestick
+$pf-chart-candelstick--data--stroke--Width: 1;
+$pf-chart-candelstick--data--stroke--Color: $pf-color-black-900;
+$pf-chart-candelstick--candle--positive--Color: $pf-color-white;
+$pf-chart-candelstick--candle--negative--Color: $pf-color-black-900;
+
+// Chart
+$pf-chart-chart--BorderWidth: 1 solid;
+
+// Donut Threshold Chart
+$pf-chart-donut--threshold--background--color-scale: ($pf-color-black-200, $pf-color-black-300, $pf-color-black-400);
+$pf-chart-donut--threshold--indicator--color-scale: ($pf-chart-color-blue-300, $pf-chart-color-gold-400, $pf-color-red-100);
```

Comment:
Thinking we should just remove all color scale vars. Just define the colors.

In react, we need to create an array, so it will likely be easier to work with individual colors. For a blue theme; for example, we merge blue colors on top of this color scale.